pdr_backend/pdr_dashboard/callbacks/callbacks_home.py

```diff
@@ -85,7 +137,8 @@ def get_display_data_from_db(
 
     @app.callback(
         Output("predictoors_table", "data", allow_duplicate=True),
-        Output("predictoors_table", "selected_rows"),
+        Output("predictoors_table", "selected_rows", allow_duplicate=True),
+        Output("table-rows-count-predictoors_table", "children", allow_duplicate=True),
```

Comment:
I don't think we should add more outputs. Why not add a callback that has predictoors_table -> data as an input, that updates the count label?